### PR TITLE
Fix/quay namespace mismatch

### PR DESCRIPTION
**Pull request by jordigilh**
* Pins Quay CR to `quay` namespace.
* ~~Updates quay subscription to `stable-3.6`.~~
* Pins `quay-setup` job and related role and rolebinding to the `rfe` namespace.

@sabre1041 @nasx let me know what you think about this.

### CODE REVIEW

File: charts/quay/templates/rfe-automation-rolebinding-quay.yaml [hunk 1]
```diff
@@ -4,12 +4,14 @@ metadata:
...
+  namespace: {{ .Values.quayRegistryCR.targetNamespace }}
+  annotations:
+    {{- toYaml .Values.setupJob.annotations | nindent 4 }}  
 roleRef:
   apiGroup: rbac.authorization.k8s.io
   kind: Role
   name: rfe-automation-quay
 subjects:
   - kind: ServiceAccount
     name: rfe-automation
```
Comment (sabre1041) [inline charts/quay/templates/rfe-automation-rolebinding-quay.yaml:16]: This should be extracted to a variable

**Review by sabre1041 — CHANGES_REQUESTED** [review]
One small change and then we should be good

File: charts/quay/templates/rfe-automation-rolebinding-quay.yaml (hunk 1 above)
Comment (jordigilh) [reply charts/quay/templates/rfe-automation-rolebinding-quay.yaml:16]: ack

Comment (jordigilh) [reply charts/quay/templates/rfe-automation-rolebinding-quay.yaml]: Done

**Comment by jordigilh** [comment]
I think this one is ready to be merged. I am getting some consistency in the deployment, barring any network issue from ansible to fetch libraries.

```
apiVersion: argoproj.io/v1alpha1
kind: Application
metadata:
  name: quay
  namespace: openshift-gitops
spec:
  destination:
    namespace: openshift-operators
    server: https://kubernetes.default.svc
  project: default
  source:
    path: charts/quay
    repoURL: https://github.com/jordigilh/rhel-edge-automation-arch.git
    targetRevision: fix/quay_namespace_mismatch
  syncPolicy:
    automated:
      prune: true
      selfHeal: true
    syncOptions:
      - CreateNamespace=true
```
In case you want to test it first.

**Review by sabre1041 — APPROVED** [review]
LGTM

**Pull request merged**